Update galileo dani

DaniDelHoyo opened the pull request:
Configuration changes and functions to create and schedule scipion projects redirected to the correct scripts (create.py and schedule.py).

danihr committed:
Configuration changes.
Calls to create.py and schedule.py for creating and scheduling scipion projects are redirected to the new locations.

danihr committed:
Configuration changes.
Calls to create.py and schedule.py for creating and scheduling scipion projects are redirected to the new locations.

pconesa commented on EMadmin/src/manage.py:
```diff
@@ -7,8 +7,8 @@
     # for using the environment specific settings file.
-    #os.environ.setdefault("DJANGO_SETTINGS_MODULE", "EMadmin.settings.development")
-    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "EMadmin.settings.production")
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "EMadmin.settings.development")
```
No puedes enviar esto co development activo. Tiene que estar como antes.

pconesa reviewed (changes requested):
Habría que cambiar el modo a produccion (ahora has puesto activado development)
Y ya puestos, eliminaria del control de codigo todo lo ue hay dentro de .idea

danihr committed:
Configuration changes.
Calls to create.py and schedule.py for creating and scheduling scipion projects are redirected to the new locations.
